contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,775 @@
+
+LogRocket does not offer as many ways to share information with external parties, though does still provide details role-based permissioning for users across multiple projects.  
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>LogRocket</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>User permissions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Control user roles and levels of access</td>
+        </tr>
+        <tr>
+            <td><strong>Public dashboards</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center">Self-host only</td>
+            <td>Make dashboards available without login</td>
+        </tr>
+        <tr>
+            <td><strong>Dashboard subscriptions</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive regular alerts about new data</td>
+        </tr>
+        <tr>
+            <td><strong>Embeddable dashboards</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Embed dashboards in other pages</td>
+        </tr>
+        <tr>
+            <td><strong>Slack</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>Discord</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>MS Teams</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
```

Comment:
This table is broken.

Also, does LogRocket support webhooks? If so, Slack etc. will be a thing.